docker/README.md

```diff
@@ -108,6 +104,8 @@ unitycatalog-cli   0.2.0-SNAPSHOT   52502d16934f   About a minute ago   1.48GB
 
 > [!NOTE]
 > In case you run into the following build exception, restart the CLI docker image build.
+> This is likely due to Java 22 used to build the image (and execute the sbt build). 
```

Comment:
Minor nit, but why call out Java 22 in this scenario since it isn't supported?

Reply:
That's what this image has always been built with, so I accepted its ups and downs. Since you brought this up, I will downgrade to the highest supported version Java 17.

Reply:
It should be good now. I had to use JDK 17 (not JRE) for sbt that gives a slightly larger image, but we're "fully-compliant" ❤️

Reply:
Merci beaucoup!